src/partitioning.c

```diff
@@ -67,8 +85,26 @@ partitioning_func_get(const char *schema, const char *funcname)
 	return func;
 }
 
+regproc
+lookup_proc(const char *schema, const char *funcname, Oid argtype, Oid *rettype)
+{
+	return lookup_proc_filtered(schema, funcname, argtype, rettype, NULL);
```

Comment:
It looks like `coltype` is only used when `filter != NULL` in `lookup_proc_filtered` so `argtype` here is useless. Should the `filter` here check that `argtype ` is correct, or should it not be an argument to `lookup_proc`?

Reply:
Actually, we could just remove this function altogether.